shells/backends/gcp_tf_backend/src/driver.py

```diff
@@ -60,38 +59,39 @@ def get_inventory(self, context):
         resource.add_sub_resource('1', port1)
         return resource.create_autoload_details()
         '''
-        gcp_service = self._can_connect(context)
-        if not gcp_service:
-            raise ValueError("Can't connect to GCP")
+        logger = LoggingSessionContext(context)
+        try:
+            gcp_service = self._can_conntect_to_gcp(context, logger)
+            if not gcp_service:
+                self._raise_and_log(logger, "There was an issue accessing GCP, please check authentication credentials.")  
+            # raise ValueError("Can't connect to GCP")
+        except Exception as e:
+            self._raise_and_log(logger, f"There was an issue initialization GCP provider resource. {e}")    
```

Comment:
Please dont use _raise_and_log method when handling exceptions.
Or if using it to handle exceptions need to change this method to re-raise the original exception.

Reply:
ok, changed